index.js

```diff
@@ -117,6 +118,7 @@ module.exports = {
         'react/jsx-uses-vars': 2,
         'react/no-deprecated': 2,
         'react/no-direct-mutation-state': 2,
+        'react/no-mutation-props': 2,
```

Comment:
(same here - adding rules to the recommended set is semver-major, so let's remove this)